yarn-project/sequencer-client/src/block_builder/solo_block_builder.ts

```diff
@@ -362,7 +362,7 @@ export class SoloBlockBuilder implements BlockBuilder {
       const { size } = await this.db.getTreeInfo(treeId);
       // TODO: Check for off-by-one errors
```

Comment:
Do you know who created this todo?

Reply:
git blame points to @spalladino

Reply:
I believe I wrote this during the London offsite in my 2nd week on the job, heh

Reply:
> I believe I wrote this during the London offsite in my 2nd week on the job, heh

@spalladino Ok ok, will nuke it in my PR up the stack

Reply:
I see that the comment 2 lines above is also stale. Will nuke that as well.